src/main/java/com/redhat/cloud/policies/app/auth/IncomingRequestFilter.java

```diff
@@ -136,9 +136,10 @@ private XRhIdentity determineXRhIdentity(String xrhid_header) {
         // Basic sanity check
         XRhIdentity rhIdentity = xrhid.get();
         if (rhIdentity.getUsername() == null || rhIdentity.getUsername().isEmpty() ||
-                rhIdentity.identity.accountNumber == null || rhIdentity.identity.accountNumber.isEmpty()
+                rhIdentity.identity.accountNumber == null || rhIdentity.identity.accountNumber.isEmpty() ||
```

Comment:
Should we remove this check? IIRC some users won't have an account id

Reply:
Done